src/components/exchange/ExchangeFloatingPanels.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { View } from 'react-native';
+import { Box, Stack } from '@rainbow-me/design-system';
+
+const ExchangeFloatingPanels = React.forwardRef<View>((props, ref) => {
+  const children = props.children!;
```

Comment:
Is it possible to avoid this non-null assertion?

Reply:
Yeah it is, just pushed an update